rust/toolchain.bzl

```diff
@@ -778,6 +780,14 @@ rust_toolchain = rule(
             ),
             mandatory = True,
         ),
+        "strip_level": attr.string_dict(
+            doc = "Rustc strip levels.",
```

Comment:
strip might be more appropriate but there is three levels.

Reply:
@matte1 would you mind applying this or doing something to add this link for users?